### PR TITLE
Test the connector with only the combined endpoint configured

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

File: src/test/java/org/osiam/client/OsiamConnectorIT.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("/context.xml")
+@TestExecutionListeners({ DependencyInjectionTestExecutionListener.class,
+        DbUnitTestExecutionListener.class })
+@DatabaseSetup("/database_seed_groups.xml")
+@DatabaseTearDown(value = "/database_tear_down.xml", type = DatabaseOperation.DELETE_ALL)
+public class OsiamConnectorIT extends AbstractIntegrationTestBase {
+
+    @Test
+    public void when_only_set_the_combined_endpoint_both_endpoints_are_valid() {
```
Comment (wallner) [inline src/test/java/org/osiam/client/OsiamConnectorIT.java]: This test name doesn't make sense. I would suggest one of the following:
- `when_combined_endpoint_is_set_both_endpoints_are_valid()`
- `both_endpoints_valid_when_combined_endpoint_ist_set()`

**Comment by wallner** [comment]
I like it, just rename the method.

**Comment by dacrome** [comment]
Updated

**Comment by wallner** [comment]
Excellent! Thanks a lot!

**Pull request merged**